packages/console/src/assets/docs/guides/web-sveltekit/README.mdx

```diff
@@ -154,8 +146,7 @@ Then use these actions in your Svelte component:
 
 To display the user's information, you can inject the `locals.user` object into the layout, thus making it available to all pages:
 
-```ts
-// +layout.server.ts
+```ts title="src/routes/+layout.server.ts"
```

Comment:
could you move "Display user info" to after "Checkpoint"?

Reply:
updated